dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -700,8 +690,6 @@ public User getUserMetadata(User user) {
         } catch (IOException ex) {
             LOG.info("Could not find user information for user " + user.getUsername());
```

Comment:
Was like this, but I'm on a mission to log all exceptions and I would also make this LOG.error

Reply:
It is a judgement call, but I think this doesn't rise to the severity required for an error. 
Logged exception